### PR TITLE
Adds sharpness to swords

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
It's a travesty that katanas currently do not work for dismemberment

### CODE REVIEW

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -68,6 +67,7 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	block_chance = 50
+	sharpness = IS_SHARP
```
Comment (xxalpha) [inline code/game/objects/items/weapons/weaponry.dm]: accurate please

Comment (Shapsy) [inline code/game/objects/items/weapons/weaponry.dm]: Have you ever cut a loaf of bread with a katana?

**Comment by ShadowDeath6** [comment]
Glorious nippon steel

**Comment by Incoming5643** [comment]
Swords will fucking cut you wide open

**Comment by Supermichael777** [comment]
BANZIIIII

**Comment by Bawhoppen** [comment]
@Incoming5643 Swords will fucking cut your hands off

File: code/game/objects/items/weapons/weaponry.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/objects/items/weapons/weaponry.dm]: Isn't accurate only used for surgery?

Comment (xxalpha) [inline code/game/objects/items/weapons/weaponry.dm]: Doesn't it influence meat cutting or am I wrong?

Also samurai surgeons.

**Comment by octareenroon91** [comment]
Pretty sure accurate matters for butchery

**Comment by Shapsy** [comment]
It's only used for cutting bread as far as I can tell

**Comment by ike709** [comment]
https://www.youtube.com/watch?v=LlsgR4gVhEc

**Comment by Zilenan91** [comment]
Do broken bottles have sharpness

**Comment by TechnoAlchemisto** [comment]
don't glass shards have is_sharp

**Comment by KorPhaeron** [comment]
Whoops

**Pull request closed without merging**